### PR TITLE
Make sfdx-cli version available in templates

### LINKED ISSUE #3: Add sfdx-cli version in the header of the generated files
In order to keep track on the version used.

### PR DESCRIPTION
**Pull request by benjamin-ross**
feat(sfdx-generator.ts): Make sfdx-cli version available in templates

fix #3

### CODE REVIEW

**Comment by benjamin-ross** [comment]
Minutes parameters now return number instead of string. 

Workaround for flag '--noprompt' type 'flag;' that returned string.

image

File: src/generator/sfdx-generator.ts [hunk 1]
```diff
@@ -194,7 +197,13 @@ export class Generator {
...
 
   private addTemplateHelper(element: Object): any {
-    return _.extend({}, element, this.templateHelpers);
+    return _.extend({ cliVersion: this.cliVersion }, element, this.templateHelpers);
+  }
+
+  private async initializeCliVersion() {
+    const unformattedCliVersion = await this.runCommand("--version");
+    // Format by removing carriage returns
+    this.cliVersion = unformattedCliVersion.replace(/\n/g, "");
```
Comment (louis-bompart) [inline src/generator/sfdx-generator.ts:206]: If I recall, `sfdx --version` is shooting a string like `sfdx-cli/6.37.0-2d5d7e1522 (windows-x64) node-v8.9.4` is that too much ?

Comment (benjamin-ross) [reply src/generator/sfdx-generator.ts:206]: Yes, the version will output a similar string, but I prefer letting the users parse this the way they want in their templates.

**Review by louis-bompart — APPROVED** [review]

**Pull request merged**